update

Hailey-Ross opened the pull request.

LordBobRoss commented:
Definitely not the final correct way to implement this, but its what I wrote up on the bus the morning. Tested on my Ubuntu Server

Redblueflame commented:
I will look into that tomorrow

kicker22004 committed:
Create Thankyou.sh

This should be remade with loops and arrays as the list gets longer. But for now it's important to make sure all helps are acknowledged. Thank you to everyone!

Commit:
The needed update, sorry it's taken so long!

Commit:
Merge branch 'master' of https://github.com/kicker22004/ECO_LINUX_SERVER_MANAGER

Commit:
Added permissions

Commit:
hotfix

Commit:
updated

Commit:
updated

Commit:
Helps when you actually give it a menu title lol.
